app/uk/gov/hmrc/ngrpropertylinkingfrontend/config/AppConfig.scala

```diff
@@ -56,6 +57,7 @@ class FrontendAppConfig @Inject()(config: Configuration, servicesConfig: Service
   override val callbackEndpointTarget: String = getString("upscan.callback-endpoint")
   override val customCurrentDate: Option[String] = config.getOptional("custom-current-date")
   override val uploadRedirectTargetBase: String = getString("upscan.upload-redirect-target-base")
+  override val ngrNotify: String = getString("microservice.services.ngr-notify.host")
```

Comment:
We are required to use `sc.baseUrl("ngr-notify")`.
This approach ensures that the base URL is constructed using the protocol, port, and host settings configured for each environment.

For reference, a similar implementation can be seen in the following example:
https://github.com/hmrc/ngr-physical-frontend/pull/81/files